src/displayapp/fonts/fonts.json

```diff
@@ -7,7 +7,7 @@
          },
          {
             "file": "FontAwesome5-Solid+Brands+Regular.woff",
-            "range": "0xf294, 0xf242, 0xf54b, 0xf21e, 0xf1e6, 0xf017, 0xf129, 0xf03a, 0xf185, 0xf560, 0xf001, 0xf3fd, 0xf1fc, 0xf45d, 0xf59f, 0xf5a0, 0xf027, 0xf028, 0xf6a9, 0xf04b, 0xf04c, 0xf048, 0xf051, 0xf095, 0xf3dd, 0xf04d, 0xf2f2, 0xf024, 0xf252, 0xf569, 0xf06e, 0xf015, 0xf00c, 0xf743"
```

Comment:
f743 removed by accident?

Reply:
It's probably an accident, yes. The string value list format is horrible to get right. Could we fix this to a normal JSON list were every entry gets its own row (for git)?

Thanks for the find 😊

Reply:
Yeah agree, the list is horrible diff wise and there's no space to explain what each symbol code actually is. I get confused all the time when I'm resolving conflicts